proto/process/agent.proto

```diff
@@ -145,6 +145,7 @@ message ECSTask {
 	repeated string ecsTags = 18;
 	repeated string containerInstanceTags = 19;
 	Host host = 20; // Used post-resolution
+	string containerInstanceArn = 21;
```

Comment:
All tasks collected by the same agent should share the same containerInstanceArn, so we can add it in CollectorECSTask

Reply:
It seems a bit tedious to surface the container instance arn to the `CollectorECSTask` 😬